apps/api/src/app/events/usecases/trigger-event-to-all/trigger-event-to-all.usecase.ts

```diff
@@ -42,18 +41,18 @@ export class TriggerEventToAll {
   }
 
   private async trigger(command: TriggerEventToAllCommand, list: SubscriberEntity[]) {
```

Comment:
@djabarovgeorge shouldn't we also move the logic in the execute method to make sure that finding the subscriber happens in the job and not here?

Reply:
So that the trigger-handler during the fanout will find all relevant subscribers, like it does with the `topic` ?

Reply:
I thought about it but then we will need to add additional logic to the trigger-handler, like passing to it a flag or something that will let it know that we need to broadcast, therefore I was not sure if we would want to do it in this task. let me know wdyt
@scopsy

Reply:
I think that to truly decouple this and make sure that the bulk trigger responds quick to the user we will need to do that. 👌

Reply:
Done